src/components/NotFoundCard/EagleAnimation.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import styled from 'styled-components'
+import { TweenLite } from 'gsap/TweenMax'
+
+export default class EagleAnimation extends React.Component {
+  componentDidMount() {
+    window.addEventListener('mousemove', this.onMouseMove)
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('mousemove', this.onMouseMove)
+  }
+
+  onMouseMove = event => {
+    const svg = document.querySelector('#svg')
+
+    const mouse = svg.createSVGPoint()
+    const leftEye = this.createEye('#left-eye')
+    const rightEye = this.createEye('#right-eye')
+
+    mouse.x = event.clientX
+    mouse.y = event.clientY
+    requestAnimationFrame(function() {
+      const point = mouse.matrixTransform(svg.getScreenCTM().inverse())
+      leftEye.rotateTo(point)
+      rightEye.rotateTo(point)
+    })
+  }
+
+  createEye = selector => {
+    const element = document.querySelector(selector)
+
+    TweenLite.set(element, {
+      transformOrigin: 'center',
+    })
+
+    const bbox = element.getBBox()
+    const centerX = bbox.x + bbox.width / 2
+    const centerY = bbox.y + bbox.height / 2
+
+    function rotateTo(point) {
+      const dx = point.x - centerX
+      const dy = point.y - centerY
+
+      const angle = Math.atan2(dy, dx)
+
+      TweenLite.to(element, 0.3, {
+        rotation: angle + '_rad_short',
+      })
+    }
+
+    return {
+      element: element,
+      rotateTo: rotateTo,
+    }
+  }
+
+  render() {
+    return (
+      <SvgWrapper>
+        <div id="tweenMaxScript" />
+        <Svg id="svg" viewBox="0 0 300 300">
```

Comment:
Probably best to use a less generic id for this.

What do you think about using React refs to get these svg elements, rather than using `querySelector()`?